src/OpenTelemetry.Contrib.Instrumentation.Wcf/README.md

```diff
@@ -20,6 +20,29 @@ using var openTelemetry = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+Instrumentation can be configured via options overload for 
+`AddWcfInstrumentation` method:
+
+```csharp
+using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+    .AddWcfInstrumentation(options => 
+    {
+        options.SuppressDownstreamInstrumentation = false;  
+
+        // Enable enriching an activity after it is created.
+        options.Enrich = (activity,eventName,message) =>    
```

Comment:
"message" is not used in this example anywhere. I'd suggest to have it used in the example. If it were not for the message, then ActivityProcessor is the recommendation for enriching any activity.

see example: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore#enrich

Reply:
Updated to a better code example which reflects usage of message.  
When enriching a WCF activity, you definitely want to have the message available.
@cijothomas Mind having another look?